nerdlandbot/helpers/constants.py

```diff
@@ -16,3 +16,14 @@
 
 POLL_MAX_TIMEOUT = 60 #minutes
 POLL_DEFAULT_TIMEOUT = 5 #minutes
+
+# the space_launches constants used
+THE_SPACE_DEVS_BASE_URL = 'https://ll.thespacedevs.com'
+THE_SPACE_DEVS_VERSION = '2.0.0'
+THE_SPACE_DEVS_UPCOMING_LAUNCH_RESOURCE = ['launch','upcoming']
+THE_SPACE_DEVS_LIMIT_TO_10_RESULTS = '?limit=10&offset=0'
```

Comment:
Maybe I missed it, but I don't think this variable is actually being used somewhere?

Reply:
Strictly it's not required because the default call returns 10 results, 
I have used it now to make sure I do get 10 results, no less, no more.